ui/widget.js

```diff
@@ -514,6 +514,16 @@ $.Widget.prototype = {
 			}
 		}
 
+		this._on( options.element, {
+			"remove": function( event ) {
+				$.each( that.classesElementLookup, function( key, value ) {
+					if ( $.inArray( event.target ) ) {
```

Comment:
`.get()` is just doing the `slice` on the jQuery.object: https://github.com/jquery/jquery/blob/3.1.0/src/core.js#L73. `$( array )` is only a little more involved as it passes through `jQuery.makeArray`.